.github/workflows/osx_startup.yaml

```diff
@@ -0,0 +1,33 @@
+name: OSX
+on: [push, pull_request]
+jobs:
+
+  test:
+    name: Startup test
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        path: 'galaxy root'
+    - uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Cache pip dir
+      uses: actions/cache@v1
+      id: pip-cache
+      with:
+        path: ~/Library/Caches/pip
+        key: pip-cache-osx-${{ matrix.python-version }}-${{ hashFiles('galaxy root/requirements.txt') }}
+    - name: Install tox
+      run: pip install tox
+    - name: fetch migrations
```

Comment:
I'd keep the "Galaxy under Conda Python" one, given the migrations are also tested in the CircleCI build, as in https://github.com/galaxyproject/galaxy/pull/9518 .